api/v1alpha1/types_etcd.go

```diff
@@ -444,7 +444,7 @@ func (e *Etcd) GetConfigmapName() string {
 
 // GetCompactionJobName returns the compaction job name for the Etcd.
 func (e *Etcd) GetCompactionJobName() string {
-	return fmt.Sprintf("%s-compact-job", string(e.UID[:6]))
+	return fmt.Sprintf("compact-job-%s", string(e.UID[:6]))
```

Comment:
If we're changing the compaction job name in this PR, I would vote to change it simply to `etcd-<role>-compaction-job`. We don't really need the etcd UID here since it doesn't add any information that can ease a human operator's life. Rather, calling it `etcd-main-compaction-job` makes more sense since:
1. It directly references the `etcd-main` instance, making it easier to locate the corresponding Etcd resource
2. It uses `etcd-` prefix, allowing us to group all etcd pods together when viewing pods in the control plane of the shoot cluster

PS: each Etcd instance can have utmost 1 compaction job running at any given time, so there would be no naming conflicts, same as today with the usage of the etcd UID rather than name.

Reply:
Done